src/utils/misc/wait.ts

```diff
@@ -1,3 +1,11 @@
-export function wait(time?: number) {
-  return new Promise<void>(resolve => setTimeout(() => resolve(), time))
+import type {Options} from '../../options'
+
+export function wait(
+  time: number,
+  advanceTimers: Exclude<Options['advanceTimers'], undefined>,
+) {
```

Comment:
If we handle timing here, we should accept the Config and take the properties we need.
```suggestion
export function wait(
  config: Config,
) {
```